kclvm/query/src/override.rs

```diff
@@ -257,7 +298,7 @@ impl<'ctx> MutSelfMutWalker<'ctx> for OverrideTransformer {
     }
 
     fn walk_assign_stmt(&mut self, assign_stmt: &'ctx mut ast::AssignStmt) {
-        if let ast::Expr::Schema(_) = &assign_stmt.value.node {
+        if let ast::Expr::Schema(_) | ast::Expr::Config(_) = &assign_stmt.value.node {
```

Comment:
Will we support normal literal assignment statements? For example
```python
myswicth = False
```
overrided to
```python
myswicth = True
```